clusterloader2/testing/load/service.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{.Name}}
+{{if IsEven 1}}
```

Comment:
`{{if IsEven .Index}}`?

Reply:
Long-term yes. But if I would do it now, I would basically remove half of existing services from kube-proxy - I don't want to do this now.

Reply:
Ok.